demo/enable-theme.groovy

```diff
@@ -24,31 +24,11 @@ if (!darkLogin) {
 
 // Login Theme
 def loginThemeConfig = Jenkins.instance.getExtensionList(SimplePageDecorator.class).get(LoginTheme.class)
```

Comment:
is this needed anymore? its built in now?

Reply:
> its built in now?

Once we switch to the plugin, yes